crates/ruff_db/src/diagnostic/render/full.rs

```diff
@@ -703,52 +704,7 @@ print()
         env.show_fix_status(true);
         env.fix_applicability(Applicability::DisplayOnly);
 
-        insta::assert_snapshot!(env.render_diagnostics(&diagnostics), @r"
-        error[unused-import][*]: `os` imported but unused
-         --> notebook.ipynb:cell 1:2:8
-          |
-        1 | # cell 1
-        2 | import os
-          |        ^^
-          |
-        help: Remove unused import: `os`
-         ::: cell 1
-        1 | # cell 1
-          - import os
-
-        error[unused-import][*]: `math` imported but unused
-         --> notebook.ipynb:cell 2:2:8
-          |
-        1 | # cell 2
-        2 | import math
-          |        ^^^^
-        3 |
-        4 | print('hello world')
-          |
-        help: Remove unused import: `math`
-         ::: cell 2
-        1 | # cell 2
-          - import math
-        2 | 
-        3 | print('hello world')
-
-        error[unused-variable][*]: Local variable `x` is assigned to but never used
-         --> notebook.ipynb:cell 3:4:5
-          |
-        2 | def foo():
-        3 |     print()
-        4 |     x = 1
-          |     ^
-          |
-        help: Remove assignment to unused variable `x`
-         ::: cell 3
-        1 | # cell 3
-        2 | def foo():
-        3 |     print()
-          -     x = 1
-        4 | 
-        note: This is an unsafe fix and may change runtime behavior
-        ");
+        insta::assert_snapshot!(env.render_diagnostics(&diagnostics));
```

Comment:
I made those tests "normal" snapshot tests because some snapshot lines contain trailing whitespace that Zed strips (might be a bug in Zed as I don't think an editor should strip trailing whitespace in a string!). 

I spent some time yesterday trying to fix the trailing whitespace but without success.